website/blog/2021-02-08-webdriverio-v7-released.md

```diff
@@ -0,0 +1,182 @@
+---
+title: WebdriverIO v7 Released
+author: Christian Bromann
+authorURL: http://twitter.com/bromann
+authorImageURL: https://s.gravatar.com/avatar/d98b16d7c93d15865f34a225dd4b1254?s=80
+---
+
+It's the time of the year were the WebdriverIO project is releasing a new major update. As it is almost a tradition we again have been rewriting the complete code base to continue further grow the project. When we [announced the v5 release](/blog/2018/12/19/webdriverio-v5-released.html) we moved from a multi-repository setup to a mono-repo. This time the rewrite of the code base is as much important and impactful but almost comes with no implications for the end user. As more and more contributor have joined the project we've noticed that using pure JavaScript can be helpful to keep the entry barrier for contributions low but ultimatively decreased the quality of contributions overall. With the growing size of code in the project keeping up with all the different types that were thrown around was even for us core contributor more and more difficult. As we already had a lot of TypeScript fans among the core contributors the decision to move to TypeScript was made quickly after meeting at the [OpenJS Collaborator Summit](https://youtu.be/HqIstZSsCTA).
+
+Our hope is that by moving to TypeScript we will introduce much less bugs continueing developing the framework. It will help improve the quality of code contributions and the speed of development of certain features. It also brings more confidence in new versions that we ship to the user.
```

Comment:
'Our hope is that by moving to TypeScript we will introduce much less bugs continueing developing the framework'
->
'Our hope is that by moving to TypeScript, fewer bugs will be introduced during continued development on the framework'